### ISSUE TITLE
Abort Job -> Error

### ISSUE THREAD

**Issue by pape-io** [cider-ci/cider-ci#48]
When we abort big jobs like tests on location app, we always get presented an error page (500) from the UI, but the abort itself actually works out.

**Comment by DrTom** [comment]
Yes, we have seen those, too. If it is a big pain to you guys we will investigate it. Otherwise we might just wait until this part of UI will be replaced. This is however more of a longterm thing.

**Comment by pape-io** [comment]
We can wait. Just wanted to have it logged. You already know the reason for it? I might post some logs later on.
